ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -61,14 +67,31 @@ def process_message(self, message: str):
         Log.debug(f'Received the message from message bus: {message}')
         try:
             EventAnalyzer(message.decode('utf-8'))
-        except Exception as err:
-            Log.error(f'Failed to analyze the event: {err}')
+            return CONSUMER_STATUS.SUCCESS
+        except ConsulException as e:
+            Log.error(f"consule exception {e} {traceback.format_exc()} for {message}. Ack Message.")
+            return CONSUMER_STATUS.SUCCESS
+        except ConfError as e:
+            Log.error(f"config exception {e} {traceback.format_exc()} for {message}. Ack Message.")
+            return CONSUMER_STATUS.SUCCESS
+        except EventFilterException as e:
+            Log.error(f"Filter exception {e} {traceback.format_exc()} for {message}. Ack Message.")
+            return CONSUMER_STATUS.SUCCESS
+        except EventParserException as e:
+            Log.error(f"Parser exception {e} {traceback.format_exc()} for {message}.  Ack Message.")
+            return CONSUMER_STATUS.SUCCESS
+        except SubscriberException as e:
+            Log.error(f"Subscriber exception {e} {traceback.format_exc()} for {message}, retry without ack.")
+            return CONSUMER_STATUS.FAILED
+        except Exception as e:
+            Log.error(f"Unknown Exception caught {e} {traceback.format_exc()}")
+            Log.error(f"Forcefully ack failed msg: {message}")
```

Comment:
@shaileshsvc should we return SUCCESS here?
is it  better not to retry this message since we don't know what has caused this failure ? This will also be in sync with earlier code at line 67 that was deleted

Reply:
As discussed returning success from here.